src/microbe_stage/MicrobeCamera.cs

```diff
@@ -119,6 +119,9 @@ public override void _Ready()
 
         materialToUpdate = (ShaderMaterial)material;
 
+        OnDisplayBackgroundParticlesChanged(Settings.Instance.DisplayBackgroundParticles);
```

Comment:
Is the length of the variable name "OnDisplayBackgroundParticlesChanged" an issue?
An alternative is replace background with decor.

Reply:
Is it? Doesn't look too bad to me.